recipes/nbcommands/meta.yaml

```
@@ -0,0 +1,84 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "nbcommands" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed.
+  url: https://github.com/vinayak-mehta/nbcommands/archive/v0.3.2.tar.gz
+  sha256: 77c6911e7171a2cabb584159e898a7932e90613512b0cfec8744055a4d3b1985
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # skip: True  # [py<36]
+  entry_points:
+    - nbblack = nbcommands._black:_black
+    - nbtouch = nbcommands._touch:touch
+    - nbgrep = nbcommands._grep:grep
+    - nbhead = nbcommands._head:head
+    - nbtail = nbcommands._tail:tail
+    - nbcat = nbcommands._cat:cat
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    # - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
```

I think you also need these dependencies, am I right?
```suggestion
    - python
    - click
    - colorama
    - nbformat
    - pigments
```

---

Sorry for not adding them earlier! Updated them.